frontend/src/scenes/trends/persons-modal/PersonsModalV2.tsx

```diff
@@ -107,8 +107,8 @@ function PersonsModalV2({ url: _url, urlsIndex, urls, title, onAfterClose }: Per
                             <span>
                                 {actorsResponse?.next ? 'More than ' : ''}
                                 <b>
-                                    {actorsResponse?.total_count} unique{' '}
-                                    {actorsResponse?.total_count === 1 ? actorLabel.singular : actorLabel.plural}
+                                    {actors.length || 'No'} unique{' '}
```

Comment:
there's a `pluralize` function that could be used here https://github.com/PostHog/posthog/blob/master/frontend/src/lib/utils.tsx#L1132

Reply:
I actually wondered about that function... Not sure if it is actually simpler than the more straightforward tertiary but happy to swap it over for now